types/multiselect/multiselect-tests.ts

```diff
@@ -0,0 +1,34 @@
+import { Options, AddOption } from 'multiselect';
```

Comment:
Add test for global use as well as module (do not use import statement in global test) 
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

Reply:
Thank you for the review.
Removed unnecessary declare global.